### DISCUSSION TITLE
Next big feature poll

### DISCUSSION THREAD

**Discussion by seerge** [seerge/g-helper#318] [Polls]
Hello, just wanted to run a poll to gather opinions of what next big feature would be most helpful

### Nvidia overclocking / XG Mobile fan / Audi visualizer - are already implemented and live in the app.

**Comment by Zackeroo** [comment]
It's a MUST for GHelper to have multiple fan profiles per performance mode. Not sure if the XG Mobile fan curve editor will do this, but any improvement to the fan curves would be excellent. At the moment, Atrofac still provides more customisation in terms of fan curve editing.

**Reply by seerge** [reply]
XG Mobile fan editor - is for dedicated gpu that you connect to your laptop :) it's something else

**Reply by Zackeroo** [reply]
Ah alright, thank you for the reply! I did a Google search when I saw that option in this poll but wasn't able to find anything on it.

**Comment by tzukiwy** [comment]
Ximic from Reddit here - would like to see if GPU Clocks and Offsets can be saved as per profile instead of applying to all 3 profiles. Hopefully others feel the same and would like to see it too!

**Reply by seerge** [reply]
You may try https://github.com/seerge/g-helper/releases/tag/v0.59

**Reply by tzukiwy** [reply]
thank you!

**Reply by Metalogic999** [reply]
@seerge this works, thanks! Very useful, then I can save the "beast mode" offsets for just Turbo mode :-)

**Comment by Metalogic999** [comment]
Would love to see an Anime Matrix audio visualizer / equalizer - the only two things I really want Anime Matrix for are that (great for parties) and the clock.

**Reply by seerge** [reply]
[GHelper.zip](https://github.com/seerge/g-helper/files/11508985/GHelper.zip)
You can check this test version

**Reply by Metalogic999** [reply]
@seerge Excellent 🥇

**Comment by Moorole** [comment]
as you brought gpu tuning options I dont know how difficult it would be to include voltagge curve adjust so it would fully replace msi afterburner which is somehow taking 3 percent cpu usage when running in my case. 

Just find it amazing how you guys are improving or replacing software made by billion dollar companies.... lol

**Reply by seerge** [reply]
@Metalogic999 hello, i have added gpu clocks cause it's more or less safe (it works via nvidia very own API) to do. As for undervolting - it's not that easy, and i don't have any experience in that area :) so i would say it's safer to use afterburner or more specialized apps that exist over years for that

**Reply by tzukiwy** [reply]
would you be able to increase the GPU Memory to at least +350 as a lot of the overclock values for the GPU memory hold around +300

**Reply by TomBonnot** [reply]
Most games and app can’t run with that much of an oc on laptops. As I have tried, games like Apex Legends or other games go full on crazy at 180 memory clock offset and then slowly dies lol

**Comment by seerge** [comment]
Anime Matrix Audio Visualizer added in https://github.com/seerge/g-helper/releases/tag/v0.64 :)

**Reply by Metalogic999** [reply]
Awesome! Will try to test this out soon

**Comment by oM0NG0o** [comment]
Seerge, 
Been digging around again, Found all this if it helps. The registry values are written and deleted by Crate on startup. BUT:
There weird way of bouncing things around I have found all this. For Hkey editing.

INITIATES HKEY:

[Init]
init_00=2
init_01=3
init_02=1
init_03=14
init_04=4

FOUND AT:
"C:\ProgramData\ASUS\CustomHotkey\initcfg.ini"

HKey Settings:
 
[Hotkey]
hk_00_opc=2
hk_00_arg=
hk_01_opc=3
hk_01_arg=
hk_02_opc=1
hk_02_arg=
hk_03_opc=14
hk_03_arg=
hk_04_opc=4
hk_04_arg=
keyconf_count_h=5
keyconf_count_v=0

FOUND AT:
"C:\Users\MONGO\AppData\Local\ASUS\CustomHotkey\CustomHotkeyConf.ini"

[Hotkey]
hk_00_opc=2
hk_00_arg=
hk_01_opc=3
hk_01_arg=
hk_02_opc=1
hk_02_arg=
hk_03_opc=14
hk_03_arg=
hk_04_opc=4
hk_04_arg=
keyconf_count_h=5
keyconf_count_v=0

Customized Hkeys:

[Hotkey]
hk_00_opc=30000
hk_00_arg=CPU-Z Application
hk_01_opc=3
hk_01_arg=
hk_02_opc=1
hk_02_arg=
hk_03_opc=14
hk_03_arg=
hk_04_opc=30004
hk_04_arg=Command Prompt
keyconf_count_h=5
keyconf_count_v=0
[LaunchApp_00]
Exists=1
LaunchCommand=C:\cpuz_x64.exe
Arguments=
AppName=CPU-Z Application
AUMID=
[LaunchApp_04]
Exists=1
LaunchCommand=C:\Command Prompt
Arguments=
AppName=Command Prompt
AUMID=

FOUND/Saved AT:
"C:\Users\MONGO\AppData\Local\ASUS\CustomHotkey\CustomHotkeyConf.ini"

I Assume 3 Hkey Board would use this same weird redundant setup. You get the drift. So you can add or adjust accordingly.

**Reply by seerge** [reply]
I have found a way to bypass and ditch asus optimization service completely. Now you can rebind M3 as well (if you stop service) https://github.com/seerge/g-helper/releases/tag/v0.65

I may add M1/M2 bindings soon too

**Comment by mara-zhu** [comment]
Have you and your team considered developing console software to replace MSI Center? As you know, MSI Center is not very easy to use, but its BIOS can be adjusted quite well.
